R/Readline/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Readline"
+version = v"8.0"
+
+# Collection of sources required to build Readline
+sources = [
+    "https://ftp.gnu.org/gnu/readline/readline-$(version.major).$(version.minor).tar.gz" =>
+    "e339f51971478d369f8a053a330a190781acb9864cf4c541060f12078948e461",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/readline-*/
+
+# Ncurses_jll provides the wide variant of the library,
+# create a link to the base variant
+ln -s ncursesw ${prefix}/include/ncurses
+export CPPFLAGS="-I${prefix}/include"
+
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-curses
+make -j${nproc} SHLIB_LIBS=-lncurses
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [p for p in supported_platforms() if !(p isa Windows)]
```

Comment:
I didn't even try to build for Windows, but I think this is usually not needed for this platform?

Reply:
Might as well skip it for now, unless someone finds they need it. Lua doesn't seem to use readline on Windows, at least.